templates/company/test/company.js

```diff
@@ -105,7 +123,15 @@ contract('Company', ([_, owner, holder1, holder2]) => {
               tokenReceipt = instanceReceipt
             } else if (creationStyle === 'separate') {
               tokenReceipt = await template.newToken(TOKEN_NAME, TOKEN_SYMBOL, { from: owner })
-              instanceReceipt = await template.newInstance(daoID, HOLDERS, STAKES, VOTING_SETTINGS, DEFAULT_FINANCE_PERIOD, useAgentAsVault, { from: owner })
+              if (installPayroll) {
+                const dummyPayrollFeed = template.address
+                const employeeManager = useOwnerAsEmployeeManager ? owner : ZERO_ADDRESS
+                const payrollSettings = [PAYROLL_DENOMINATION_TOKEN, dummyPayrollFeed, PAYROLL_RATE_EXPIRY_TIME, employeeManager]
+                instanceReceipt = await newInstance(daoID, HOLDERS, STAKES, VOTING_SETTINGS, DEFAULT_FINANCE_PERIOD, useAgentAsVault, payrollSettings)
+              }
+              else {
```

Comment:
Super style nitpick, we generally put `else`s on the trailing bracket

Prefer

```js
if {
} else {
}
```

Over

```js
if {
}
else {
}
```